### PR TITLE
fix(cmd/immuclient): improve immuclient tx error message

### LINKED ISSUE #711: improve immuclient tx error messages
**What happened**
immuclient supports to query by transaction id
`immuclient tx #`

But the error messages are not clear or when the transactions id is invalid:
`immuclient tx 0` returns `illegal arguments`, `immuclient tx -0` tries to map it with a flag `unknown shorthand flag`

**What you expected to happen**
better error message would be `invalid TXid - transaction id range start with 1`

### PR DESCRIPTION
**Pull request by mangalaman93**
Fixes #711 

I don't think we should do something about the case `-0`.
Reference: https://github.com/spf13/cobra/issues/124

### CODE REVIEW

**Comment by jeroiraz** [comment]
thanks @mangalaman93

Could you add the validation in `VerifiedGetTxByID` as well?

File: cmd/immuclient/immuc/getcommands.go [hunk 1]
```diff
@@ -31,6 +31,9 @@ func (i *immuc) GetTxByID(args []string) (string, error) {
 	if err != nil {
 		return "", fmt.Errorf(" \"%v\" is not a valid id number", args[0])
 	}
+	if id == 0 {
+		return "", fmt.Errorf(" invalid tx id (should be bigger than 0)")
```
Comment (jeroiraz) [inline cmd/immuclient/immuc/getcommands.go:35]: First white space may not be needed

Comment (mangalaman93) [reply cmd/immuclient/immuc/getcommands.go]: Updated the message. I think equality is fine, we have an uint64.

**Comment by mangalaman93** [comment]
Done

**Review by jeroiraz — APPROVED** [review]

**Pull request merged**